src/main/java/org/frc5687/rapidreact/Constants.java

```diff
@@ -28,16 +28,12 @@
 
         public static final double DEADBAND = 0.2;
 
-        public static final double MAX_MPS = 3.5; // Max speed of robot (m/s) .
+        public static final double MAX_MPS = 1.5; // Max speed of robot (m/s) .
 
-<<<<<<< Updated upstream
-        public static final double MAX_ANG_VEL =
-                Math.PI * 2.0; // Max rotation rate of robot (rads/s)
+        public static final double MAX_ANG_VEL = Math.PI * 1.5; // Max rotation rate of robot (rads/s)
         public static final double MAX_MPSS = 0.5; // Max acceleration of robot (m/s^2)
```

Comment:
Only affects auto, not teleop.

Reply:
Can we find a way to be able to adjust the accel for teleop? If it's possible during auto it should be possible during teleop.